mbed_greentea/mbed_greentea_cli.py

```diff
@@ -250,6 +299,18 @@ def main():
                     action="store_true",
                     help='List available binaries')
 
+    parser.add_option('', '--list-test-cases',
+                    dest='list_test_cases',
+                    default=False,
+                    action="store_true",
+                    help='List available test cases inside the binaries')
+
+    parser.add_option('', '--list-test-plan',
+                    dest='list_test_plan',
+                    default=False,
+                    action="store_true",
+                    help='List implemented, not implemented, and conflicting test cases from the specified Test Plan (--test-plan)')
+
```

Comment:
This is an optional idea: instead of introducing two new options for listing things, maybe we could use the existing the flags? For instance, `--list-test-cases` could be `--list -v`. And `--list-test-plan` could be `--list -v` when `--test-plan` is specified.

Just an idea though! Our list of parameters is getting pretty long, so it'd be nice to reuse these if at all possible.